internal/engine/engine.go

```diff
@@ -729,7 +730,7 @@ func InitializeOperatorChecks(ctx context.Context, p policy.Policy, cfg Operator
 		return []check.Check{
 			operatorpol.NewScorecardBasicSpecCheck(operatorsdk.New(cfg.ScorecardImage, exec.Command), cfg.ScorecardNamespace, cfg.ScorecardServiceAccount, cfg.Kubeconfig, cfg.ScorecardWaitTime),
 			operatorpol.NewScorecardOlmSuiteCheck(operatorsdk.New(cfg.ScorecardImage, exec.Command), cfg.ScorecardNamespace, cfg.ScorecardServiceAccount, cfg.Kubeconfig, cfg.ScorecardWaitTime),
-			operatorpol.NewDeployableByOlmCheck(cfg.IndexImage, cfg.DockerConfig, cfg.Channel),
+			operatorpol.NewDeployableByOlmCheck(cfg.IndexImage, cfg.DockerConfig, cfg.Channel, cfg.CSVTimeout),
```

Comment:
This should probably become a functional Option, especially if we foresee any other things needing to be exposed.

Reply:
This is good feedback, add `opts` for this instead.